common/trinary/flex_trit.c

```diff
@@ -98,8 +98,17 @@ size_t flex_trits_insert(flex_trit_t *const to_flex_trits, size_t const to_len,
 #if defined(FLEX_TRIT_ENCODING_1_TRIT_PER_BYTE)
   memcpy(to_flex_trits + start, flex_trits, num_trits);
 #else
-  for (size_t i = 0; i < num_trits; i++) {
-    flex_trits_set_at(to_flex_trits, to_len, start + i, flex_trits_at(flex_trits, len, i));
+  if (0 == (start % NUM_TRITS_PER_FLEX_TRIT)) {
```

Comment:
Can we not always "handle head" so we get "new" start index and always memcpy from there?

Reply:
actually no :(